runtime/src/iree/builtins/ukernel/pack.c

```diff
@@ -94,11 +95,18 @@ static void iree_uk_pack_using_tile_func(const iree_uk_pack_params_t* params,
   bool l1_has_padding = outer_size1 * tile_size1 != params->in_size1;
   iree_uk_ssize_t l0_full_tile_end = outer_size0 - (l0_has_padding ? 1 : 0);
   iree_uk_ssize_t l1_full_tile_end = outer_size1 - (l1_has_padding ? 1 : 0);
+
+  // TODO (Issue #11632): this fix-up is needed only because at the moment,
```

Comment:
```suggestion
  // TODO(#11632): this fix-up is needed only because at the moment,
```

The `TODO(#ISSUENUMBER)` syntax is special and picked up by editors/etc.